src/plugins/console/public/application/models/sense_editor/integration.test.js

```diff
@@ -948,8 +948,6 @@ describe('Integration', () => {
         autoCompleteSet: [
           tt('field1.1.1', { f: 1 }, 'string'),
           tt('field1.1.2', { f: 1 }, 'string'),
-          tt('field2.1.1', { f: 1 }, 'string'),
-          tt('field2.1.2', { f: 1 }, 'string'),
         ],
```

Comment:
changes in this test files is the reverse of changes in this PR https://github.com/elastic/kibana/pull/162917